docs/source/app_developers_guide/ubuntu.rst

```diff
@@ -114,6 +114,13 @@ stable or nightly.  We recommend using the stable repository.
 
       user@validator$ sudo apt-get install -y sawtooth
 
+#. Install the Sawtooth DevMode consensus engine package. Run the following
```

Comment:
The rest of the Sawtooth documentation uses "dev mode" as two words (without caps unless at the start of a sentence or in a heading).  However, this is a minor nitpick.

Reply:
P.S. I noticed that the info message below says "Devmode 1.0", so one could make the case for writing "Devmode" as one word, as long as one doesn't capitalize the "M".

Reply:
I changed it to `Devmode` as suggested. It will restart the 10 quatzillion hour build, but it still needs another reviewer in any case.